tools/pythonpkg/src/pandas/scan.cpp

```diff
@@ -141,6 +146,20 @@ double PandasScanFunction::PandasProgress(ClientContext &context, const Function
 	return percentage;
 }
 
+void PandasScanFunction::PandasBackendScanSwitch(PandasColumnBindData &bind_data, idx_t count, idx_t offset,
+                                                 Vector &out) {
+	auto backend = bind_data.numpy_col->Backend();
+	switch (backend) {
+	case PandasColumnBackend::NUMPY: {
+		NumpyScan::Scan(bind_data, count, offset, out);
+		break;
+	}
```

Comment:
I think eventually we may want to do that, because currently we deal with mixed dataframes by converting them to pyarrow, then to duckdb.
We likely want to eventually get rid of the conversion to pyarrow first, as pyarrow -> duckdb isn't entirely zero-copy